pymongo/encryption.py

```diff
@@ -23,6 +23,7 @@
 from typing import (
     TYPE_CHECKING,
     Any,
+    Dict,
```

Comment:
Should this be lowercase `dict` for consistency across type hints?

Reply:
This was the error I ran into, which left me just to leave it. 
```
typecheck: commands[1]> mypy --install-types --non-interactive bson gridfs tools pymongo
pymongo/encryption.py:86: error: "dict" is not subscriptable  [misc]
        document_class=dict[str, Any], uuid_representation=STANDARD
                       ^
Found 1 error in 1 file (checked 86 source files)
```

Reply:
We're importing future `__annotations__`, so I would expect that to work, hmm.